msal_extensions/windows.py

```diff
@@ -39,6 +39,15 @@ def raw(self):
         _MEMCPY(blob_buffer, pb_data, cb_data)
         return blob_buffer.raw
 
+_err_description = {
```

Comment:
Do we have some knowledge on WinError 0 (https://github.com/Azure/azure-cli/issues/20278)?

Reply:
I don't know. The "error 0" is particularly strange. The zero is supposed to mean success.

Regardless, like I [admitted here](https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/108/files#r801311953), these error description are still more like troubleshooting trace. I should probably change this PR's title to "Provide catch-able exceptions for all dpapi errors". Eventually Azure CLI might still need to catch them and then tell end user "just turn off encryption by configuring ~/.azure/...", which this MSAL EX package won't provide anyway.